### ISSUE TITLE
utc_offset is deprecated as of November 2019 

### ISSUE THREAD

**Issue by vonec** [MadimetjaShika/vuetify-google-autocomplete#97]
utc_offset is deprecated as of November 2019 and will beturned off in November 2020. Use utc_offset_minutes instead.

this needs to be fixed

**Comment by wrtisans** [comment]
any update in this ?

**Comment by Daniyal83** [comment]
Yes. That deprecation message is annoying. Would you fix that, please?
